static/third/bootstrap/css/bootstrap.css

```diff
@@ -810,7 +810,7 @@ address {
 code,
 pre {
   padding: 0 3px 2px;
-  font-family: Monaco, Menlo, Consolas, "Courier New", monospace;
+  font-family: Monaco, Menlo, Consolas, "Liberation Mono", "Courier New", monospace;
```

Comment:
Do we need to modify the bootstrap copy?  We're aiming to completely unfork Bootstrap, and I'd expect our own copy (above) to take precedence.